tests/src/tracing/eventpipe_onoff/eventpipe_onoff.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Threading.Tasks;
+using System.IO;
+
+using EventPipe;
+
+namespace eventpipe_onoff
+{
+    class EventPipeSmoke
+    {
+        private static int allocIterations = 10000;
+        private static int trivialSize = 0x100000;
+        private static bool keepOutput = false;
+
+        static int Main(string[] args)
+        {
+            string outputFilename = System.IO.Path.GetTempPath() + Guid.NewGuid().ToString() + ".netperf";
+            TraceConfiguration config = CreateConfiguration(outputFilename);
+
+            Console.WriteLine("\tStart: Enable tracing.");
+            TraceControl.Enable(config);
```

Comment:
I think you can call TraceControl.EnableDefault here and then you don't need the CreateConfiguration method below.